### PR TITLE
Removes Styptic Powder, Silver Sulfadiazine and Synthflesh from base Medical Hyposprays

### PR DESCRIPTION
**Pull request by SchrodingersWolf**
## What Does This PR Do
As title. Removed those reagents from the list of chemicals accepted by the base Medical Hyposprays. (The ones you get from NanoMed Pluses)

## Why It's Good For The Game
Back before #21722 was merged, injecting someone with styptic powder/silver sulfadiazine would actually heal the injected person of brute/burn damage as the injection is not considered an ingestion somehow. It made sense to put it in safe_chem_list back then, even though it was a weird quirk.

Now that #21722 is merged though, styptic powder/silver sulfadizine does 15 toxins when you inject 30u into someone. And since the base hypospray shouldn't accept chemicals that could cause harm without being emagged, I feel like it is time for those two chems to go.

Removed Synthflesh from the list as well because it does absolutely nothing if it isn't a touch reaction. Sure, it isn't causing toxins or anything, but I'm removing it for consistency's sake, as well as to make it to reduce a bit of confusion.

## Images of changes
Styptic
SilverSulf
Synthflesh

## Testing
Loaded up private server, tried adding the chemicals to a Medical Hypospray from NanoMed Plus. Realised that the hypospray ate my hard-earned chems.

## Changelog
tweak: You can no longer add Styptic Powder, Silver Sulfadiazine or Synthflesh to Medical Hyposprays from NanoMed Plus
/:cl:

### CODE REVIEW

**Review by Henri215 — APPROVED** [review]
LGTM

**Review by lewcc — APPROVED** [review]
Code looks good

File: code/modules/reagents/chemistry/machinery/chem_master.dm [hunk 1]
```diff
@@ -28,10 +28,10 @@
...
 	var/static/list/pill_bottle_wrappers
 	var/static/list/bottle_styles
-	var/list/safe_chem_list = list("antihol", "charcoal", "epinephrine", "insulin", "teporone", "silver_sulfadiazine", "salbutamol",
-									"omnizine", "stimulants", "synaptizine", "potass_iodide", "oculine", "mannitol", "styptic_powder",
-									"spaceacillin", "salglu_solution", "sal_acid", "cryoxadone", "blood", "synthflesh", "hydrocodone",
-									"mitocholide", "rezadone", "menthol")
+	var/list/safe_chem_list = list("antihol", "charcoal", "epinephrine", "insulin", "teporone", "salbutamol", "omnizine",
+									"stimulants", "synaptizine", "potass_iodide", "oculine", "mannitol", "spaceacillin",
+									"salglu_solution", "sal_acid", "cryoxadone", "blood", "hydrocodone","mitocholide",
+									"rezadone", "menthol")
```
Comment (DGamerL) [inline code/modules/reagents/chemistry/machinery/chem_master.dm:34]: Wrong edited file, the actual list is in `hypospray.dm` for hyposprays

Comment (SchrodingersWolf) [reply code/modules/reagents/chemistry/machinery/chem_master.dm:34]: SEAN!!!!!!

**Pull request merged**